src/patternfly/components/Content/content.scss

```diff
@@ -77,7 +77,7 @@
   --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
   --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
```

Comment:
The correct variable format is `--pf-global--ListStyle`. This should be fixed in _variables.scss.